### PR TITLE
Add initial Vercel configuration files

### PR DESCRIPTION
**Pull request by ajschmidt8**
We will soon be switching our hosting from Netlify to Vercel.

This PR adds the initial files necessary to start testing on Vercel.

### CODE REVIEW

**Comment by ajschmidt8** [comment]
Once this PR merges, all future PRs will deploy with both Netlify and Vercel until we've completed the migration to Vercel.

**Comment by ajschmidt8** [comment]
> @ajschmidt8 is attempting to deploy a commit to the **rapidsai** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=rapidsai&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%222f785a682f2f3847ead12307a06457379ad359b4%22%7D%2C%22id%22%3A%22QmQKPon1depEx4T5pwjPkr1ALyKHFY7C9M9X7vikfr12JY%22%2C%22org%22%3A%22rapidsai%22%2C%22prId%22%3A401%2C%22repo%22%3A%22rapids.ai%22%7D).

I've disabled this "protection" using the docs here: https://vercel.com/docs/deployments/git/vercel-for-github#deployment-authorizations-for-forks.

This isn't necessary for us since we don't use any sensitive environment variables for our deploys.

**Review by bdice — APPROVED** [review]
Approving so you have a starting point to work on further improvements with CI.

**Pull request merged**
